src/core/include/cesium/omniverse/Tokens.h

```diff
@@ -225,7 +227,7 @@ const omni::fabric::Type inputs_alpha_cutoff(omni::fabric::BaseDataType::eFloat,
 const omni::fabric::Type inputs_alpha_mode(omni::fabric::BaseDataType::eInt, 1, 0, omni::fabric::AttributeRole::eNone);
 const omni::fabric::Type inputs_base_alpha(omni::fabric::BaseDataType::eFloat, 1, 0, omni::fabric::AttributeRole::eNone);
 const omni::fabric::Type inputs_base_color_factor(omni::fabric::BaseDataType::eFloat, 3, 0, omni::fabric::AttributeRole::eColor);
-const omni::fabric::Type inputs_debug_color(omni::fabric::BaseDataType::eFloat, 3, 0, omni::fabric::AttributeRole::eColor);
+const omni::fabric::Type inputs_tile_color(omni::fabric::BaseDataType::eFloat, 4, 0, omni::fabric::AttributeRole::eNone);
```

Comment:
Why does the `AttributeRole` change?

Reply:
My understanding (which could be wrong) is that `eColor` corresponds to the `color` MDL type, whereas `eNone` is a better fit for the `float4` MDL type. The `color` type doesn't seem to support alpha.

Reply:
Also I haven't noticed any color space differences between `eColor` and `eNone`. Maybe linear is assumed for both, which works in our favor.